ov/firmware_drivers.go

```diff
@@ -145,6 +147,30 @@ func (c *OVClient) GetFirmwareBaselineById(id string) (FirmwareDrivers, error) {
 	return firmwareId, nil
 }
 
+func (c *OVClient) GetFirmwareBaselineByNameandVersion(name string) (FirmwareDrivers, error) {
+	fwNameVersion := strings.SplitAfter(name, ",")
+	if len(fwNameVersion) < 2 {
+		return FirmwareDrivers{}, errors.New("provide firmware name and version separated by comma")
+	}
+	fwname, version := fwNameVersion[0], fwNameVersion[1]
```

Comment:
i think in our docs, we need to specify clearly that first part in the name is the name and second part is version